.travis.yml

```diff
@@ -113,6 +113,9 @@ matrix:
             # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
             - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
           script: true
+        - os: linux
+          compiler: gcc
+          env: CONFIGURE_TARGET="linux-generic32" CONFIG_OPTS="--strict-warnings no-shared no-dso no-pic no-autoload-config no-engine no-comp no-cms no-dh no-dsa no-ec2m no-srp no-des no-mdc2 no-md4 no-rc2 no-rc4 no-bf no-camellia no-cast no-idea no-rmd160 no-seed no-whirlpool no-ocsp no-ocb no-sm2 no-sm3 no-sm4 no-ssl3 no-ssl3-method no-psk no-heartbeats no-nextprotoneg no-sctp no-srtp no-ct no-async no-asm -DOPENSSL_SMALL_FOOTPRINT"
```

Comment:
I am surprised that you have gone for "no-ec2m" instead of "no-ec". The "no-ec" option is the one that breaks most frequently out of all the options we have - so I would expect to see that in there.

Reply:
ec2m was broken with OpenSSL_1_0_2t if I remember correctly

Reply:
I meant 1.1.0l, the last release in the 1.1.0 stable,
see a3b54f0f5de1ad17889fd23aee7c230eefc300cd for instance

Reply:
Well - we can't test both in a single build. But for no-ec I have at least: #7620, #6437, #6321, #6253, #5997, #5673, #5664, #5618, #5418, #5149, #4981, #4874, #4518, #4369, #4134, #3817, #3693, #3503, #3430, #3321, #2858, #2624, #2564, #2547, #2157, #2192, #2153, and others beyond that...

Reply:
no-ec2m breaks by comparison do still occur - but much less frequently.

Reply:
:-)

ec-2m is really not used, anywhere I could think of, while ec is getting more important
all the time.

Maybe I can add the no-ec2m thing to another build, would give the same coverage.

Reply:
Yes - that would work